BrowserKit/Sources/WebEngine/WKWebview/WKEngine.swift

```diff
@@ -77,4 +78,40 @@ public class WKEngine: Engine {
         timer.resume()
         shutdownWebServerTimer = timer
     }
+
+    // MARK: - Clearing data
+
+    public func clearCaches() {
+        DiskReaderModeCache.shared.clear()
+        MemoryReaderModeCache.shared.clear()
+
+        let dataTypes = Set([WKWebsiteDataTypeDiskCache, WKWebsiteDataTypeMemoryCache])
+        WKWebsiteDataStore.default().removeData(ofTypes: dataTypes, modifiedSince: .distantPast, completionHandler: {})
+    }
+
+    public func clearCookies() {
```

Comment:
Just curious is this cookie clearance related to reader mode? or just move it as part of the PR. I ask because of the cookie bug and I was not aware of us handling outside of the settings clearance

Reply:
I added this API so we can keep the settings clearance as it is today. It relates to reader mode with:
```
       DiskReaderModeCache.shared.clear()
       MemoryReaderModeCache.shared.clear()
```
And I added the other data clearance since I don't want to forget about them 😆 Too many things to migrate to WebEngine